pwa/src/components/PatientInfoForm.tsx

```diff
@@ -0,0 +1,265 @@
+
+  // Data for Radio Buttons
+  const sexValues = [
+    { value: 0, label: "Male" },
+    { value: 1, label: "Female" },
+  ];
+
+  const radioYesNo = [
+    { value: 0, label: "No" },
+    { value: 1, label: "Yes" },
+  ];
+
+  /* istanbul ignore next */
+  const onSubmit = () => {
+    // This shows how to access the values of the form
+    const firstNameField: HTMLInputElement = document.getElementById(
+      "firstname"
+    ) as HTMLInputElement;
+    const lastNameField: HTMLInputElement = document.getElementById(
+      "lastname"
+    ) as HTMLInputElement;
+    const nicknameField: HTMLInputElement = document.getElementById(
+      "nickname"
+    ) as HTMLInputElement;
+    const birthdateField: HTMLInputElement = document.getElementById(
+      "birthdate"
+    ) as HTMLInputElement;
+    const complaintField1: HTMLInputElement = document.getElementById(
+      "complaint1"
+    ) as HTMLInputElement;
+    const complaintField2: HTMLInputElement = document.getElementById(
+      "complaint2"
+    ) as HTMLInputElement;
+    const complaintField3: HTMLInputElement = document.getElementById(
+      "complaint3"
+    ) as HTMLInputElement;
+
+    console.log("First name:", firstNameField.value);
+    console.log("Last name:", lastNameField.value);
+    console.log("Nickname:", nicknameField.value);
+    console.log("Birth Date:", birthdateField.value);
+    console.log(
+      "Complaints: ",
+      complaintField1.value +
+        " " +
+        complaintField2.value +
+        " " +
+        complaintField3.value
+    );
+    console.log("Wants Planning?: " + planning);
+  };
+
+  return (
+    <div id="patient-info-wrapper">
+      <TextareaItem
+        title="First Name"
+        placeholder="First Name"
+        autoHeight
+        id="firstname"
+      />
+
+      <TextareaItem
+        title="Last Name"
+        placeholder="Last Name"
+        id="lastname"
+        autoHeight
+      />
+
+      <TextareaItem
+        title="Nickname"
+        placeholder="Nickname"
+        id="nickname"
+        autoHeight
+      />
+
+      <List renderHeader={() => "Sex"}>
+        {sexValues.map((i) => (
+          <RadioItem
+            data-testid={`${i.label.toLowerCase()}-button`}
+            id={`${i.label.toLowerCase()}-button`}
+            key={i.value}
+            name="sex"
+            checked={sex === i.value}
+            onClick={() => setSex(i.value)}
+          >
+            {i.label}
+          </RadioItem>
+        ))}
+      </List>
+
+      <WhiteSpace />
+
+      <InputItem placeholder="YYYY-MM-DD" id="birthdate">
+        Birth Date
+      </InputItem>
+
+      <InputItem type="number" placeholder="Age" id="age">
+        Age{" "}
+      </InputItem>
+
+      <InputItem type="number" placeholder="Phone" id="phone">
+        Phone
+      </InputItem>
+
+      <TextareaItem title="Town" placeholder="Town" id="town" autoHeight />
+
+      <List renderHeader={() => "Visit?"}>
```

Comment:
What does `Visit` mean? We have the opportunity to be more verbose, as we aren't limited to the size of a green card.
I would imagine this means "Has the patient had previous visits with CHI?"

Reply:
There is a database field `first_visit` which seems to match your assumption. I will update the label to be "First Visit?".

Reply:
Go ahead @meghanmc and link the commit in a comment here. If the assumption is wrong we can correct later.

Reply:
Resolved in [this commit](https://github.com/Haiti-Distributed-Healthcare-System/hdhs/pull/66/commits/d2ebfa3f0df60e1d94620ced121c28d0b4b7b8bb)